docs/getting_started/folders_and_files/derivatives.md

```diff
@@ -199,31 +201,47 @@ my_dataset/
     analysis/
   sub-01/
   ...
+  dataset_description.json
 ```
 
-A BIDS Derivatives dataset may contain references to its input datasets
-in the `sourcedata/` subdirectory:
+Disadvantage is that such organization would complicate distribution of the raw BIDS dataset
+by itself as it would require explicit exclusion of datasets within its `derivatives/` folder.
```

Comment:
I'm not following this sentence.

Reply:
Ah, now I understand. I think I would do this:

```suggestion
```

Reply:
The *full sentence* could be 

```suggestion
The disadvantage of storing derivatives under `derivatives/` folder of the raw BIDS dataset 
is that such organization would complicate distribution of the raw BIDS dataset
by itself as it would require explicit exclusion of datasets within its `derivatives/` folder.
```

Does that convey the message?